CHANGELOG.md

```diff
@@ -6,6 +6,40 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.2.0] 2019-10-21
+
+### Added
+
+- Added a new [public plugin interface](pkg/secretless/plugin) for building connector plugins
+- Added a new [public log interface](pkg/secretless/log) for standardizing logging
+- Added reporting on code coverage data to unit test output
+- Added ability to run k8s-demo test on GKE
+
+### Changed
+
+- Refactored existing connectors to use new public connector plugin interface
+- Changed the core proxy and plugin manager to support the new public connector
+  plugin interface
+- Edited website Google Group links to link to Discourse
+- Updated the [example plugin](test/plugin) to implement the new plugin interface
+- Minor format changes to Apache 2.0 license
+- Project structure reorganized
+- Internal code updated to use v2 config instead of v1 config
+- Goreleaser build updated to cross-compile linux and darwin
+- Updated Conjur tests to use official CLI image
+
+### Fixed
+
+- Improve namespace cleanup in k8s-ci/test
+- Add COMPOSE_PROJECT_NAME to tests to fix namespace collision errors
+- Updated k8s demo to use LoadBalancer on Services to avoid NodePort conflicts
+- Clarified quick demo directions
+- Improved error-handling / retry logic in k8s-ci
+
+### Deprecated
+
+- `Protocol` key in v2 config is replaced with `connector` key
```

Comment:
It's not replaced yet afaik

Reply:
`protocol` is not used by the docs on prod anymore or in our tests that use v2 config. these changes were made in #826 and cyberark/secretless-docs#185